docs_src/camera_serial_number.md

```diff
@@ -1,24 +1,32 @@
-# How to get RealSense camera serial number
+# Get Serial Number of Intel® RealSense™ Camera
 
-- run the built docker image in interactive mode, with root, and mount the host devices
+Do the following to get the serial number of a Intel® RealSense™ Camera:
 
-`docker run --rm -u root -it --privileged sco-soc:2.0 bash`
+1. Run the built docker image in interactive mode, with root, and mount the host devices:
 
-- while in the container run `rs-enumerate-devices` to list metadata of all attached RealSense devices
+    ```
+    docker run --rm -u root -it --privileged sco-soc:2.0 bash
+    ```
 
-```
-/home/pipeline-zoo/workspace# rs-enumerate-devices
-Device info: 
-    Name                          :     Intel RealSense D435
-    Serial Number                 :----->serial number<------
-    Firmware Version              :     05.08.15.00
-    Recommended Firmware Version  :     05.13.00.50
-    Physical Port                 :     /sys/devices/pci0000:00/0000:00:14.0/usb2/2-3/2-3:1.0/video4linux/video0
-    Debug Op Code                 :     15
-    Advanced Mode                 :     YES
-    Product Id                    :     0B07
-    Camera Locked                 :     YES
-    Product Line                  :     D400
-    Asic Serial Number            :     <not the serial number>
-    Firmware Update Id            :     <not the serial number>
-```
\ No newline at end of file
+2. While in the container, run `rs-enumerate-devices` to list metadata of all attached Intel® RealSense™ products:
+
+    ```
+    /home/pipeline-zoo/workspace# rs-enumerate-devices
+    ```
+    
+    Expected result:
+    ```
+    Device info: 
+        Name                          :     Intel RealSense D435
+        Serial Number                 :----->serial number<------
+        Firmware Version              :     05.08.15.00
+        Recommended Firmware Version  :     05.13.00.50
+        Physical Port                 :     /sys/devices/pci0000:00/0000:00:14.0/usb2/2-3/2-3:1.0/video4linux/video0
+        Debug Op Code                 :     15
+        Advanced Mode                 :     YES
+        Product Id                    :     0B07
+        Camera Locked                 :     YES
+        Product Line                  :     D400
+        Asic Serial Number            :     <not the serial number>
+        Firmware Update Id            :     <not the serial number>
+    ```
```

Comment:
should we also mention something like check if your realsense camera is plugin into your system if you don't see expected result?